packages/koenig-lexical/src/plugins/FloatingFormatToolbarPlugin.jsx

```diff
@@ -123,7 +123,7 @@ function FloatingFormatToolbar({isText, editor, anchorElem, blockType, isBold, i
         if (
             selection !== null &&
             nativeSelection !== null &&
-            !nativeSelection.isCollapsed &&
+            !nativeSelection.isCollapsed() &&
```

Comment:
This one needs to stay a property as it's a native DOM selection rather than a Lexical selection 🙂

Reply:
Good call. I got a little replacement happy.